src/kernels/jupyter/remoteKernelFinder.ts

```diff
@@ -113,6 +112,7 @@ export class RemoteKernelFinder implements IRemoteKernelFinder {
                 return items;
             } catch (ex) {
                 traceError(`Error fetching remote kernels:`, ex);
+                throw ex;
```

Comment:
Previously we'd swallow exceptions and return an empty list, now we throw the error and higher up when we have errors we just fallback to the cache (this behaviour is documented in the calling code)